vm/src/bytecode.rs

```diff
@@ -37,6 +37,12 @@ impl CodeObject {
     }
 }
 
+bitflags! {
```

Comment:
I think I would go for an extra boolean in the function: `has_defaults`. So no extra packages.

Reply:
This will eventually need to carry 4 values (per https://docs.python.org/3/library/dis.html#opcode-MAKE_FUNCTION), which is why I went with the bitflags approach. It also maintains symmetry with the CPython implementation.

If you'd still prefer for me to change it, I'd be happy to do so.

(FWIW, we also already pull bitflags in due to a couple of dependencies, so this has no net impact of our dependency tree.)

Reply:
My reasons would be that using booleans would allow the rust compiler to implement bitfields by itself. I think bitfields are an optimization from times where memory was scarce and compilers did not implement this. I would not be surprised if rust under the hood implemented booleans as bitfields.

Leave it as it is for now :)